pkg/fission-cli/function.go

```diff
@@ -206,6 +207,11 @@ func fnCreate(c *cli.Context) error {
 		}
 	}
 	entrypoint := c.String("entrypoint")
+	regex := regexp.MustCompile("^[a-zA-Z0-9_.]*$")
+	if !regex.MatchString(entrypoint) {
+		log.Fatal("Entrypoint should not contain any special character except underscore('_') and dot('.').")
```

Comment:
We should add the same check at `fnUpdate` as well. 
Also, please make the check a callable function in `pkg/fission-cli/util/util.go` so that we can reuse it.

Reply:
Added a function in ```util.go``` and added the check in fnUpdate as well. Thanks for pointing out this missed fnUpdate.